.github/workflows/ruby-ci.yml

```diff
@@ -0,0 +1,40 @@
+name: Ruby CI
+
+on:
+  - push
+
+jobs:
+  build:
+    name: Ruby ${{ matrix.version }} ${{ matrix.gemfile }}
+    runs-on: ubuntu-latest
+    env:
+      BUNDLE_GEMFILE: ${{ matrix.gemfile }}
+    strategy:
+      matrix:
+        version:
+          - 2.5
+          - 2.6
+          - 2.7
+        gemfile:
+          - gemfiles/Gemfile.rails60
+          - gemfiles/Gemfile.latest-release
+          - gemfiles/Gemfile.shopify
+          - gemfiles/Gemfile.rails-edge
+        # exclude:
```

Comment:
did you leave these on purpose or just was playing around? :D

Reply:
I leaved that comment on purpose in case we need it in the future so we don't need to search for it (when we support Ruby 3 for example).

But we can remove as well, don't have a strong opinion there =)